checker/src/org/checkerframework/checker/index/upperbound/UpperBoundAnnotatedTypeFactory.java

```diff
@@ -354,6 +349,16 @@ public QualifierHierarchy createQualifierHierarchy(MultiGraphFactory factory) {
     }
 
     /**
+     * Check if type is either LTL or LTOM. Prevents duplication and makes code more readable.
```

Comment:
Rather than "check", please use more specific language, such as "Return true if".

I suggest that you delete the second sentence.  A reader will assume that you didn't add this to make the code less readable...